DVCodingStyle.md

```diff
@@ -0,0 +1,863 @@
+# lowRISC SystemVerilog Coding Style Guide for Design Verification
+
+This document is a supplemental Style Guide to the [lowRISC Verilog style
+guide](https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md),
+with emphasis on writing code for Design Verification (DV) in SystemVerilog,
+following the UVM methodology.
```

Comment:
Link the UVM methodology to: 
https://www.accellera.org/images//downloads/standards/uvm/uvm_users_guide_1.2.pdf

Reply:
[Or indicate this in the introduction]

Reply:
I think a link would look better, added.